eva-peng/week2/mon/brooklyn/css/style.css

```diff
@@ -0,0 +1,45 @@
+/* 
+
+## Style Hints: 
+
+Body Font Size = 12px
+	 Line Height = 17px
+	 Font Family = "Times New Roman", serif
+	 Font Color = #656565
+
+Link Hover Color = #20222a
+
+*/
+p {
+	font-size: 12px;
+	line-height: 17px;
+	font-family: 'Times New Roman', Times, serif;
+	font color: #656565;
```

Comment:
`font color` is not a valid property. 

that is why you can see it red squiggly lines in your file. the correct property is just `color: #656565` :)